soc/arm/nxp_imx/rt/CMakeLists.txt

```diff
@@ -1,12 +1,10 @@
 #
-# Copyright (c) 2017, NXP
+# Copyright (c) 2017-2021, NXP
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-
-zephyr_sources(
-  soc.c
-  )
+zephyr_sources_ifdef(CONFIG_SOC_SERIES_IMX_RT11XX soc_rt11xx.c)
+zephyr_sources_ifdef(CONFIG_SOC_SERIES_IMX_RT10XX soc_rt10xx.c)
```

Comment:
Please explain in the commit message why you're splitting up soc.c (i.e., significantly different clocking)

Reply:
Done